src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,72 @@
+.menu {
+  background: #d12d35;
+  z-index: 1;
```

Comment:
Consider using a variable for the background color for maintainability and consistency.